File: BarbeariaEInclusao/src/app/model/solicitacao.ts
export class Solicitacao {
    idSolicitacao: number | null;
    idBarbeiroDestinatario: number | null;
    idBarbeariaRemetente: number | null;
    estadoSolicitacao: string | null;

    constructor(init?: Partial<Solicitacao>) {
        this.idSolicitacao = init?.idSolicitacao ?? null;
        this.idBarbeiroDestinatario = init?.idBarbeiroDestinatario ?? null;
        this.idBarbeariaRemetente = init?.idBarbeariaRemetente ?? null;
        this.estadoSolicitacao = init?.estadoSolicitacao ?? null;
    }
}

File: barbeariainclusao/barbeariainclusao/barbeariainclusao/barbeariainclusao/barbeariainclusao/src/main/java/br/cefetmg/barbeariainclusao/model/Solicitacao.java
package br.cefetmg.barbeariainclusao.model;

import lombok.*;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Solicitacao {

    private Long idSolicitacao;

    // ID do barbeiro que vai receber a solicitação
    private Long idBarbeiroDestinatario;

    // ID da barbearia que enviou a solicitação
    private Long idBarbeariaRemetente;

    // Estado da solicitação: PENDENTE, ACEITO, RECUSADO
    private String estadoSolicitacao;
}
